docs/dev/run-nb-test.py

```diff
@@ -17,6 +17,7 @@
 import uuid
 
 JOBS_JSON = 'docs/dev/jobs-config.json'
```

Comment:
Rename to `BASE_JOBS_JSON` and move the file to `docs/dev/base-jobs-config.json`, then add diff JSONs

Reply:
how about

`python3 docs/dev/run-nb-test.py --cli-profile docs-ci --nbs etl/hail-interoperability --jobs-json-config hail`

`@click.option('--jobs-json-config', default='docs/dev/jobs-config.json', help='Configuration JSON file for Databricks jobs cluster.')`

`def main(cli_profile, workspace_tmp_dir, dbfs_init_script_dir, source_dir, nbs, jobs_json_config):
    JOBS_JSON = jobs_json_config`